src/assets/angular-tour.scss

```diff
@@ -0,0 +1,138 @@
+//colors
+$background_color: #1c252e;
+$white: #fff;
+$text_color: #cbd0d4;
+$button_color: #555;
+$button_hover_color: #777;
+$close_button_hover: #eee;
+$black_60: rgba(0, 0, 0, 0.6);
+
+
+%base-button {
+  background-color: $button_color;
+  transition: background-color 300ms ease-out;
+  cursor: pointer;
+  font-family: inherit;
+  font-weight: 700;
+  line-height: 1;
+  position: relative;
+  text-decoration: none;
+  text-align: center;
+  display: inline-block;
+  font-size: .8em;
```

Comment:
We should also avoid introducing too many different font-size in the app for visual consistency.   I have to admit that I haven't done very good job on this regard during the pre-user-study crunch, but we should avoid making it worse :)